resources/monitoring/profile-production.yaml

```diff
@@ -17,13 +17,29 @@ prometheus:
               storage: 20Gi
 
 prometheusOperator:
+  namespaces:
+    releaseNamespace: "true"
```

Comment:
That block should not get in as it is specific to the SAP setup and will break compatibility from OS perspective